desktop/panel/database.test.js

```diff
@@ -162,7 +185,7 @@ for (const subprocess of RUNNERS) {
                 expect(new Date(v[0].date)).toStrictEqual(
                   new Date('2021-01-01')
                 );
-              } else {
+              } else if (!t.query.indexOf('SELECT') === -1) {
```

Comment:
If there's no else section when you changed this that means there might be a test case that is silently passing. Maybe also add a final `else { throw new Error('Unexpected testing condition') } ` to this if chain?